docs: explained what node keyword does in terminal (#39020)
nodejs/node: doc/api/synopsis.md

});
```

> Save the file and go back to the terminal. Entering the following commands run the file named `hello-world.js`:

Reviewer: ```suggestion
Save the file. Then, in the terminal window, to run the `hello-world.js` file,
enter:
```

Reviewer: (See table row for `Enter` at https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui for relevant guidelines/examples on describing user interactions like this.)
